server.js

```diff
@@ -25,6 +25,7 @@ app.use(function(req, res, next) {
  * ROUTES *
```

Comment:
You wrote models, but never even included them in your server file; it's clear you weren't able to get a database working at all in this application.